server/__init__.py

```diff
@@ -156,24 +157,27 @@ async def pre_setup(self):
                 addon = app_defs.versions[addon_version]
                 addon._update_enums = self._update_enums
 
-    async def create_config_hash(
+    async def create_action_config_hash(
         self,
         identifier: str,
         context: ActionContext,
         user: UserEntity,
         variant: str,
     ) -> str:
         """Create a hash for action config store"""
-        if identifier.startswith(IDENTIFIER_PREFIX):
-            # Change identifier to only app name
-            identifier = identifier[len(IDENTIFIER_PREFIX):]
+        if not identifier.startswith(IDENTIFIER_PREFIX):
+            return await super().create_action_config_hash(
+                identifier, context, user, variant
+            )
+
+        # Change identifier to only app name and one task id
+        identifier = identifier[len(IDENTIFIER_PREFIX):]
```

Comment:
Since the server runs a recent python version - did you know you could do `identifier.removeprefix(IDENTIFIER_PREFIX)`?

Reply:
Good point, will do.